dashboard/app/config.go

```diff
@@ -167,6 +167,15 @@ type CoverageConfig struct {
 	// WebGitURI specifies where can we get the kernel file source code directly from AppEngine.
 	// It may be the Git or Gerrit compatible repo.
 	WebGitURI string
+
+	// EmailRegressionsTo species the regressions recipient.
+	// If empty, regression analysis is disabled.
+	EmailRegressionsTo string
+
+	// RegressionThreshold is a minimal basic block coverage drop in a file.
+	// The amount of files in the dir and other factors do not matter.
+	// 50 is a recommended default.
+	RegressionThreshold int
```

Comment:
Let's set it to the recommended value if not specified in the config. Then we won't need to duplicate the same value across the config for every namespace, and will be able to change it for all namespaces.